src/Workspaces/Core/MSBuild.BuildHost/MSBuild/Constants/PropertyNames.cs

```diff
@@ -9,6 +9,7 @@ internal static class PropertyNames
         public const string AllowUnsafeBlocks = nameof(AllowUnsafeBlocks);
         public const string AppConfigForCompiler = nameof(AppConfigForCompiler);
         public const string BaseAddress = nameof(BaseAddress);
+        public const string BaseIntermediateOutputPath = nameof(BaseIntermediateOutputPath);
```

Comment:
Did you add this but then never use it?

Reply:
oops - was playing around with it and left this there